### PR TITLE
Switch to zendframework/zend-coding-standard

### PR DESCRIPTION
**Pull request by Koopzington**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -18,7 +18,8 @@
     "require-dev": {
         "athletic/athletic": "~0.1",
         "phpunit/PHPUnit": "~4.0",
-        "squizlabs/php_codesniffer": "^2.6.2"
+        "squizlabs/php_codesniffer": "^2.6.2",
+        "zendframework/zend-coding-standard": "~1.0.0"
```
Comment (weierophinney) [inline composer.json]: We can remove the dependency on php_codesniffer, as it's included in the zend-coding-standard. :smile:

**Comment by webimpress** [comment]
@froschdesign maybe this should be also merged for 3.2.0 release?

**Comment by webimpress** [comment]
File `.php_cs` should be also removed and entry form `.gitattributes`

**Comment by froschdesign** [comment]
@Koopzington 
Thanks! 👍

**Pull request merged**